test/e2e/fleet_test.go

```diff
@@ -318,14 +366,16 @@ func scaleFleet(f *v1alpha1.Fleet, scale int32) (*v1alpha1.Fleet, error) {
 	return framework.AgonesClient.StableV1alpha1().Fleets(defaultNs).Patch(f.ObjectMeta.Name, types.JSONPatchType, []byte(patch))
 }
 
+const replicasCount = 3
```

Comment:
Let's move this up to the top with the rest of the consts - that's the usual go way.